clouddq/classes/rule_type.py

```diff
@@ -26,6 +26,7 @@
 FORBIDDEN_SQL = re.compile(r"[;\-#|\\/]")
 NOT_NULL_SQL = Template("$column IS NOT NULL")
 REGEX_SQL = Template("REGEXP_CONTAINS( CAST( $column  AS STRING), '$pattern' )")
+NOT_BLANK_SQL = Template("$column != ''")
```

Comment:
what do you think about using `TRIM(col) != ''` to account for blanks with multiple spaces?

Reply:
This makes sense, I will make the required changes and test it.